expandedTree/expandedTree_test.go

```diff
@@ -0,0 +1,31 @@
+package expandedTree
+
+import (
+	"log"
+	"testing"
+)
+
+func TestReadTree(t *testing.T) {
+	var actualName = "root"
+	var aFound = false
+	tree, err := ReadTree("testdata/newick.txt", "testdata/fasta.fasta")
+	treeNodes := GetTree(tree)
+	if err != nil {
+		log.Fatal("Problem in ReadTree")
```

Comment:
Test functions in gonomics have a special syntax for Fatal. Consider switching to using `t.Error()` or `t.Errorf()` instead of log.Fatal for test functions. The behavior is very similar but I believe t.Error() has some special functionality for advanced unit testing.

Reply:
This is exactly the kind of feedback I was hoping for. I haven't been exploring this part of go at all, thanks Dan!